docs/controls/WebView-known-issues.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Known Issues for WebView control for Windows Forms and WPF
+author: normesta
+description: This guide highlights known limitations with the current release of the WebView control for Windows Forms and WPF applications.
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, WebView, Windows Forms, WPF, known issues, release notes
+---
+
+# Known Issues of the WebView control for Windows Forms and WPF applications
+
+This document describes issues and limitations in this release of the **WebView** control.
+
+## WebView control members
+
+The **WebView** control does not implement these methods of the [IWebViewControl](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol).
+
+* [NavigateToLocalStreamUri](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.navigatetolocalstreamuri)
+
+* [NavigateWithHttpRequestMessage](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.navigatewithhttprequestmessage)
+
+* [CapturePreviewToStreamAsync](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.capturepreviewtostreamasync)
+
+* [CaptureSelectedContentToDataPackageAsync](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.captureselectedcontenttodatapackageasync)
+
+* [BuildLocalStreamUri](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.buildlocalstreamuri)
+
+The **WebView** control implements these events of the [IWebViewControl](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol), but they don't work as expected.
+
+* [LongRunningScriptDetected](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.longrunningscriptdetected)
+
+* [UnsafeContentWarningDisplaying](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.unsafecontentwarningdisplaying)
+
+* [UnsupportedUriSchemeIdentified](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.unsupportedurischemeidentified)
+
+* [UnviewableContentIdentified](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.unviewablecontentidentified)
+
+* [NewWindowRequested](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.newwindowrequested)
+
+## Rendering and layout
+
+* The **WebView** control doesn't appear at the correct scale when users move the host application between monitors that have different screen resolutions.
+
+* The **WebView** control doesn't appear as expected on some high-resolution displays.
```

Comment:
As long as they indicate per monitor DPI mode and use Windows 10's V2 for WinForms, this works with one exception: changing the DPI mode while the application is on the monitor that is changing DPI. In that case, we do not correctly process and scale the DPI event and there's a gap around the control to where it used to be. The workaround is to restart the application, and the DPI will scale to the monitor (even if it is moved to another monitor with a different DPI). The most common situation for this is docking/undocking a laptop.